tests/fixtures/bugs.json

```diff
@@ -51,4 +51,8 @@
@mozilla.com", "nick": "marco", "real_name": "Marco Castelluccio [:marco]", "email": "mcastelluccio@mozilla.com", "id": 420453}, {"id": 403756, "email": "mconley@mozilla.com", "real_name": "Mike Conley (:mconley) (:\u2699\ufe0f)", "nick": "mconley", "name": "mconley@mozilla.com"}, {"name": "mozilla+bmo@noorenberghe.ca", "real_name": "Matthew N. [:MattN]", "id": 305228, "email": "mozilla+bmo@noorenberghe.ca", "nick": "MattN"}, {"nick": "nika", "real_name": "Nika Layzell [:nika] (ni? for response)", "id": 534482, "email": "nika@thelayzells.com", "name": "nika@thelayzells.com"}, {"nick": "past", "email": "pastith@gmail.com", "id": 363318, "real_name": "Panos Astithas (he/him) [:past] (please ni?)", "name": "pastith@gmail.com"}, {"email": "Tobias.Marty@gmx.de", "real_name": "TMart", "id": 572549, "nick": "Tobias.Marty", "name": "Tobias.Marty@gmx.de"}], "creator_detail": {"name": "felipc@gmail.com", "real_name": ":Felipe Gomes (needinfo for replies!)", "id": 208747, "email": "felipc@gmail.com", "nick": "Felipe"}, "assigned_to": "felipc@gmail.com", "cf_cab_review": "---", "summary": "XPCOMUtils.defineLazyProxy (was: lazy listeners for the content process)", "url": "", "blocks": [1461247, 1461248, 1461444], "regressed_by": [], "cf_fx_points": "---", "id": 1457988, "whiteboard": "[fxperf:p1]", "cf_rank": null, "last_change_time": "2018-05-17T16:42:00Z", "type": "enhancement", "cf_qa_whiteboard": "", "cf_status_firefox62": "fixed", "cf_performance_impact": "---", "cf_status_firefox61": "wontfix", "cf_a11y_review_project_flag": "---", "op_sys": "Unspecified", "cf_tracking_firefox_esr102": "---", "cf_fx_iteration": "---", "cf_status_firefox110": "---", "severity": "normal", "alias": null, "target_milestone": "Firefox 62", "cf_tracking_firefox_relnote": "---", "cf_tracking_firefox111": "---", "see_also": [], "cc": ["jryans@gmail.com", "kmaglione+bmo@mozilla.com", "mail@johann-hofmann.com", "mcastelluccio@mozilla.com", "mconley@mozilla.com", "mozilla+bmo@noorenberghe.ca", "nika@thelayzells.com", "pastith@gmail.com", "Tobias.Marty@gmx.de"], "comment_count": 12, "cf_status_firefox_esr102": "---", "classification": "Client Software", "platform": "Unspecified", "is_open": false, "cf_last_resolved": "2018-05-16T10:04:21Z", "cf_status_firefox111": "---", "version": "Trunk", "cf_tracking_firefox110": "---", "dupe_of": null, "cf_crash_signature": "", "is_confirmed": true, "qa_contact": "", "keywords": [], "cf_status_firefox112": "---", "cf_has_str": "---", "cf_user_story": "", "votes": 1, "regressions": [], "status": "RESOLVED", "filed_via": "unknown", "cf_tracking_firefox112": "---", "creation_time": "2018-04-30T18:24:10Z", "creator": "felipc@gmail.com", "duplicates": [], "resolution": "FIXED", "depends_on": [1462400], "priority": "P1", "mentors_detail": [], "flags": [], "assigned_to_detail": {"name": "felipc@gmail.com", "nick": "Felipe", "real_name": ":Felipe Gomes (needinfo for replies!)", "email": "felipc@gmail.com", "id": 208747}, "cf_webcompat_priority": "---"}
```

Comment:
Instead of adding new bugs, can we reuse existing ones? In any case, we already have plenty of bugs labelled as `0`, so there's no need for extra cases.

Reply:
> Instead of adding new bugs, can we reuse existing ones? In any case, we already have plenty of bugs labelled as `0`, so there's no need for extra cases.

Because of the condition, all the performance bug cases in `bugs.json` are skipped. That's why I added these bugs that meet the labeling criteria.

https://github.com/mozilla/bugbug/blob/cbb004045e9525fd2ae8d6707784084fd6601de9/bugbug/models/performancebug.py#L101-L106

Before the bug addition:
```
INFO     bugbug.models.performancebug:performancebug.py:109 0 performance bugs
INFO     bugbug.models.performancebug:performancebug.py:113 0 non-performance bugs
```

Reply:
Make sense. The `cf_performance_impact` field is relatively new.